cnxarchive/views/content.py

```diff
@@ -266,3 +266,41 @@ def get_extra(request):
     resp.content_type = 'application/json'
     resp.body = json.dumps(results)
     return resp
+
+
+@view_config(route_name='content-included-in', request_method='GET')
+def get_books_containing_page(request):
+    """Return a list of book names and UUIDs that contain a given module UUID."""
+    settings = get_current_registry().settings
+    routing_args = request and request.matchdict or {}
+
+    page_ident_hash = routing_args.get('page_ident_hash', '')
+    if page_ident_hash:
+        try:
+            uuid, version = split_ident_hash(page_ident_hash)
+        except IdentHashShortId as e:
+            uuid = get_uuid(e.id)
+            version = e.version
+        except IdentHashMissingVersion as e:
+            raise httpexceptions.HTTPNotFound()
+
+    results = {'books': []}
+    cur_res = []
+
+    with psycopg2.connect(settings[config.CONNECTION_STRING]) as db_connection:
+        with db_connection.cursor() as cursor:
+            cursor.execute(SQL['get-books-containing-page'], {'page_uuid': uuid, 'page_version': version})
+
+            try:
+                cur_res = cursor.fetchall()
+            except (TypeError, IndexError,):  # None returned
```

Comment:
This exception will never be hit. `cursor.fetchall()` will always return something, even an empty list if no rows are found. You'd be better suited to checking with an `if not cur_res: raise ...`.